### DISCUSSION TITLE
Add support for embedding input

### DISCUSSION THREAD

**Discussion by Andcircle** [vllm-project/vllm#6316] [Q&A]
I saw similar ask for this feature, but seems like still open-ended, not solved yet.
We're using different model e.g. SAM / SigLIP as LEGO component to build our own VLM, only language model is time consuming.
Hope we can prepare embedding input, and feed it into language model with VLLM.

Just trying to confirm, do we already have this feature? If yes, could somebody share an example code.
If not, is this feature on roadmap?

Really appreciated.

**Comment by yuanzhiyong1999** [comment]
Have you found an alternative?

**Comment by Andcircle** [comment]
Nope, keep waiting 😅
